src/services/pcn-router/test/test_shadow/test1.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+# 			    TOPOLOGY
+#	+------+				+------+
+#	|  r1  |--------------------------------|  r2  |
+#	+------+ p1:10.0.0.1	    p2:10.0.0.2 +------+
+#	         
+#
+
+set -x
+
+# Create the router r1
+polycubectl router del r1 # remove r1 if already existed
+polycubectl router add r1 shadow=true
+
+# Create the router r2
+polycubectl router del r2 # remove r2 if already existed
+polycubectl router add r2 # the router r2 can also be shadow, it is the same
+
+# Create the ports
+polycubectl r1 ports add p1 ip=10.0.0.1 netmask=255.255.255.0
+polycubectl r2 ports add p2 ip=10.0.0.2 netmask=255.255.255.0
+
+# Connect
+polycubectl connect r1:p1 r2:p2
+
+# Ping
+sudo ip netns exec pcn-r1 ping 10.0.0.2 -c 3 -i 0.5
+
+# Clean up
```

Comment:
Please do it as in other tests, create a cleanup function and then install a trap to execute it also in case of error.
https://github.com/polycube-network/polycube/blob/master/src/services/pcn-router/test/test1_3.sh